zipkin-lens/src/types/image.d.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2015-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+declare module '*.png';
```

Comment:
This is necessary to avoid errors when importing png files as shown below.

```typescript
import logoSrc from '../../img/zipkin-logo.png';
```

Reply:
Hmm that's weird, it should be working already with react-scripts.

https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/lib/react-app.d.ts

Do you know why that isn't getting picked up?

Reply:
Nice catch!
I noticed that the reference directive in react-app-env.d.ts isn't working and fixed it 🙇